src/Repository/AdherentRepository.php

```diff
@@ -546,4 +547,17 @@ public function findAdherentsUuidByEmailAddress(string $emailAddress): array
             return $uuid->toString();
         }, array_column($query->getArrayResult(), 'uuid'));
     }
+
+    public function findAllEnabledAdherents(int $offset = 0): IterableResult
+    {
+        $qb = $this->createQueryBuilder('a');
+
+        $query = $qb
+            ->andWhere('a.status = :status')
+            ->setParameter('status', Adherent::ENABLED)
+            ->setFirstResult($offset)
+            ->getQuery();
+
+        return $query->iterate();
```

Comment:
you could just chain the return i guess:
```
return $this
    ->createQueryBuilder('a')
    ->andWhere('a.status = :status')
    // ..
    ->getQuery()
    ->iterate()
;
```

Reply:
then you can chain the return from the query builder

Reply:
>I agree but to stay consistent

Consistent with what? we never did that before... I agree we should separate implementations though, but only in regard of the first line caller otherwise I don't see the benefit in case of a builder.